docs/maya/skin.md

```diff
@@ -138,10 +141,17 @@ In order to better visualize deformer constraints and attributes in the Maya vie
 To enable the debugger the *Debug* checkbox must be marked. To select the specific feature you would like to visualize, choose it from the list provided in *Features*. The features that can be visualized with the debugger in the AdnSkin deformer are:
 
  - **Hard Constraints**: For each vertex, a line will be drawn from the simulated mesh vertex to the corresponding point on the reference mesh if its *Hard Constraints* weight is greater than 0.0.
- - **Soft Constraints**: For each vertex, a line will be drawn from the simulated mesh vertex to the corresponding point on the reference mesh if its *Soft Constraints* weight is greater than 0.0.
+ - **Shape Preservation**: For each vertex with a shape preservation weight greater than 0.0, a line will be drawn from each adjacent vertex to the opposite adjacent vertex.
  - **Slide Constraints**: For each vertex, a line will be drawn from the simulated mesh vertex to the corresponding point on the reference mesh if its *Slide Constraints* weight is greater than 0.0.s
+ - **Sliding Surface On Target**: For each vertex, lines will outline the target triangles within the reach of its *Sliding Distance*.
```

Comment:
Now that we are adding "shape preservation" and "sliding surface on target" as new AdonisFX concepts would be great to add them at `terminology.md`!

Reply:
Added Shape Preservation! However I am unsure if "sliding surface on target" should be added to the terminology or where exactly would it fit in, as it is more a result of other constraints and parameters.

Reply:
That's true, it's okey. Check that in the message you say "suring simulation" instead of "during.

Reply:
Corrected!